MyApplication/app/src/main/java/com/example/myapplication/HomeActivity.kt

```diff
@@ -0,0 +1,52 @@
+package com.example.myapplication
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.example.myapplication.adapter.ItemAdapter
+import com.example.myapplication.data.Datasource
+import kotlinx.android.synthetic.main.activity_home.*
+
+class HomeActivity : AppCompatActivity() {
+   // private lateinit var posts: MutableList<Post>
+  //  val items: ArrayList<Post> = ArrayList()
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_home)
```

Comment:
Change init of activity view to view binding https://developer.android.com/topic/libraries/view-binding